app/src/main/java/app/grapheneos/camera/ActivityLifeCycleHelper.kt

```diff
@@ -0,0 +1,38 @@
+package app.grapheneos.camera
+
+import android.app.Activity
+import android.app.Application
+import android.os.Bundle
+
+class ActivityLifeCycleHelper(
```

Comment:
`LifeCycle` -> `Lifecycle`. 
`ActivityLifecycleCallbacksObserver` sounds better
Don't use lambdas for callbacks here. Create an interface with empty methods bodies as default and pass as  constructor parameter.

Reply:
> Don't use lambdas for callbacks here

why ?

Reply:
When another component will require another callbacks or all of them then you'll have to create more constructor parameters. Interface usage will cleanup code

Reply:
Also unclear semantic when `onActivityResumed` and `onActivityPaused` call same `callback.invoke(activity)`